packages/openops/src/lib/ai/providers.ts

```diff
@@ -0,0 +1,77 @@
+import { amazonBedrockProvider } from './providers/amazon-bedrock';
+import { anthropicProvider } from './providers/anthropic';
+import { azureProvider } from './providers/azure-openai';
+import { cerebrasProvider } from './providers/cerebras';
+import { cohereProvider } from './providers/cohere';
+import { deepinfraProvider } from './providers/deep-infra';
+import { deepseekProvider } from './providers/deep-seek';
+import { googleProvider } from './providers/google';
+import { groqProvider } from './providers/groq';
+import { lmntProvider } from './providers/lmnt';
+import { mistralProvider } from './providers/mistral';
+import { openAiProvider } from './providers/openai';
+import { openaiCompatibleProvider } from './providers/openai-compatible';
+import { perplexityProvider } from './providers/perplexity';
+import { togetherAiProvider } from './providers/together-ai';
+import { xaiProvider } from './providers/xai';
+
+export interface AiProvider {
+  getModels(): string[];
```

Comment:
This one does not need to be a function. It can be models: string[];

```suggestion
  models: string[];
```